daprovider/das/dasutil/dasutil.go

```diff
@@ -119,7 +119,17 @@ func (d *writerForDAS) Store(message []byte, timeout uint64) containers.PromiseI
 	go func() {
 		cert, err := d.dasWriter.Store(ctx, message, timeout)
 		if err != nil {
-			promise.ProduceError(err)
+			// If the aggregator failed due to insufficient backends, signal explicit fallback
+			// to allow batch poster to try the next DA provider in the chain. The Aggregator
+			// has a loud ERROR if the threshold of failing committee members is approaching,
+			// which should give time to correct any errors to avoid fallback. Otherwise
+			// the operator can set disable-dap-fallback-store-data-on-chain to totally
+			// disable automatic fallback to EthDA.
+			if errors.Is(err, ErrBatchToDasFailed) {
```

Comment:
I don't like that this is in code that doesn't seem to affect just anytrust.
Instead, it should be done somewhere more specific like directly in RPCAggregator.
Another option is to wrap with something like the WriterPanicWrapper, could be amended to optionally panic, optionally request Fallback.

Reply:
writerForDAS is specifically for AnyTrust so I think it's okay to do this wrapping here (DAS almost always refers to AnyTrust - I want to change it to just use AnyTrust but that will involve a lot of renaming and moving files so I don't want to do it until all of this is merged.)